src/ostorlab/agent/mixins/agent_open_telemetry_mixin.py

```diff
@@ -12,6 +13,7 @@
 
 from opentelemetry import trace
 from opentelemetry.exporter.jaeger import thrift as jaeger
+from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
```

Comment:
don't import classes directly. 
```suggestion
from opentelemetry.exporter import cloud_trace
```